packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -191,8 +204,19 @@ class ToCiceroMarkVisitor {
                             ...nodeInformation.properties,
                             `${NS_PREFIX_CommonMarkModel}Strong`,
                         ];
+                    } else if (runTimeProperties.name === 'w:color') {
+                        if (runTimeProperties.attributes['w:val'] === 'C45911') {
+                            colorCodePresent = true;
+                        }
+                    } else if (runTimeProperties.name === 'w:highlight') {
+                        if (runTimeProperties.attributes['w:val'] === 'lightGray') {
```

Comment:
[TemplateMark Dingus](https://templatemark-dingus.netlify.app/) uses `#C7254E` and `#F9F2F4` for color and highlight respectively. I think it would make sense if we are consistent.

Reply:
@algomaster99 We can change the color but for highlighting I am afraid as word supports limited highlighting colors.
Image when color is changed to `C7254E`.
![Screenshot (79)](https://user-images.githubusercontent.com/59891164/124894034-e7df8100-dff8-11eb-805f-5e04bdcf8d9d.png)

Should I change the color with respect to above image?

Reply:
Just making sure here. Highlight colour is the background colour while the simply "colour" is the font colour, right?

Reply:
MS Word itself doesn't have limited colours for font-colour so it's quite tough to understand changing color to `#C7254E` is not possible.

Reply:
> Just making sure here. Highlight colour is the background colour while the simply "colour" is the font colour, right?

Yes

Reply:
> MS Word itself doesn't have limited colours for font-colour so it's quite tough to understand changing color to `#C7254E` is not possible.

I am not sure as when I changed the color in `OOXML` to `#C7254E` it gave no errors. So I assumed that it might pe possible.

Reply:
@algomaster99 
So should we keep the colors as it is? I mean no changes in text or highlight color.

Reply:
I tried to tweak colours myself and it turns out that you can change the font color to any RGB colour however, highlight colour only takes specific values which are listed in the link I have linked below. 
```xml
<w:r>
  <w:rPr>
    <w:color w:val="C7254E" />
    <w:shd w:fill="F9F2F4"/>
  </w:rPr>
   <w:t>Inline code</w:t>
</w:r>
```
You can try to generate an OOXML like above to achieve the desired effect.

Reference: http://officeopenxml.com/WPtextShading.php.

Reply:
@algomaster99 will update the PR ASAP.

Reply:
@algomaster99 
Done